roles/addons/powerman/templates/powerman.conf.j2

```diff
@@ -14,20 +14,23 @@ include "/etc/powerman/ipmipower.dev"
   {% endfor %}
 {% else %}
   {% set range = groups['all'] %}
-{% endif %}
+{%- endif %}
```

Comment:
This `-` is not needed.

Reply:
See comment here: https://github.com/bluebanquise/bluebanquise/pull/541#discussion_r608388253